### DISCUSSION TITLE
[Scam Alert] hoho project (hoho token)

### DISCUSSION THREAD

**Discussion by micro73** [Webamoozcom/warning-list#138] [Investigation بررسی]
ادرس سایت: hoho.mobi

ادرس کانترکت: 0x3Aabcabf828e684b3D9329831453aE180bD9D2a7  - BSC 

 - ادعای پروژه : راه اندازی و تبدیل شدن به پلتفرم بازی های بلاکچینی 
 - به گفته یکی از اعضا یونیک بودن این پروژه برای این هست که میخواهد به تاس بلاکچینی تبدیل شود

مشکلات و عیب های شناسایی شده : :::::

 شبکه اجتماعی مانند (تلگرام،اینستاگرام،توییتر،دیسکورد) ندارد و در سایت اشاره ای بهش نشده است . فقط در سایت یک قسمت برای چت تهیه شده است که چت ها باید توسط ادمین ها تایید شود .

 هویت گردانندگان و تهیه کنندگان پروژه نا مشخص و مخفی است .

 نود و نه (99)درصد توکن ها در یک ادرس قرار دارد که بزرگترین هشدار است .  تعداد کل توکن  ها 60,000,000,000 HOHO است که تقریبا اکثرش ماین شده و در داخل یک ولت است  و در کل 161 هولدر دارد .

 توکن hoho  در تاریخ 23 مارچ 2022 راه اندازی شده است و خیلی زمان زیادی نیست.

 وایت پیپر پروژه خیلی سطحی نوشته شده است و به هیچ عنوان از نظر فنی صحبتی از کارکرد پروژه نکرده و فقط از دید و اینده ای که دارند صحبت شده است و اکثر صحبت ها از اطلاعات عمومی بلاکچین است .

 در کوین مارکت کپ یا کوین گکو لیست نشده است تا اطلاعات بیشتری قابل دریافت باشد .

پروژه گین هاب ندارد تا کارکرد دولوپر ها قابل مشاهده باشد .

خرید توکن داخل خود سایت و با bnb  انجام میشود .  

 سایت پروژه در تاریخ 2-3-2022 راه اندازی شده است .

 اطلاعات whois  دامنه سایت مخفی شده است و قابل مشاهده نیست . 

 توکنومیک و توکن اکونومیک ندارد. 

                                                                                   هشدار کلاهبرداری و اسکم

**Comment by micro73** [comment]
موارد بیشتر:

1. بحث زیرمجموعه گیری اضافه شده است.

![photo_2023-02-13_00-05-03](https://user-images.githubusercontent.com/103103225/218404737-5e65d999-bacd-4653-9293-1492c1364ff5.jpg)

[Plan HOHO.pdf](https://github.com/Webamoozcom/warning-list/files/10720059/Plan.HOHO.pdf)

2. قیمت توکن را به صورت فرضی اعلام کردند که 0.006 هست.

**Comment by micro73** [comment]
قابلیت‌هایی که اسمارت کانترکت این توکن دارد به شرح زیر است:

صاحب کانترکت اجازه تغییر هر زمانی کانترکت دارد یعنی می‌تواند  فروش را لغو کند
لیکوییدیتی پول ندارد
تمام ساپلای دست خود سازنده هست
امکان فریز‌کردن موجودی دارد
و حتی فراخوان دادند و موجودی فریز شده است.
امکان سوزاندن توکن در ولت هولدر هم دارد.

**Comment by micro73** [comment]
جدیدا ادرس اسمارت کانترکت به 

0x5794ce8578ed5f40b78a5a051d648b0d47256353

تغییر کرده و هنوز در bsc explore وریفای نشده که قابل بررسی باشد.

**Comment by micro73** [comment]
در صحبت‌های جدید
بر این ادعا هستند که امارات به دلیل محدودیت‌های موجود پشت پروژه هست که قابل تایید نیست.
و از هولدینگ هوهو صحبت میکنند که هیچ طالاعاتی ازش در دسترس نیست 

دروغ شاخ و دم ندارد

**Comment by micro73** [comment]
hoho.mobi uses an illegal copy of EvolutionScript. Please report this site.
در حال استفاده از اسکریپت رایگان برای سایت هستند و هزینه‌ای نکردند

**Comment by libertyir** [comment]
ظاهرا پروژه با تیم بارکس در ارتباطه . 
صرفا جهت اصلاح بند نامشخص بودن تیم. 
باقی موارد بررسی نشد.(تایید/رد نمی کنم)
https://www.shodan.io/host/185.231.59.65

![image](https://user-images.githubusercontent.com/90643392/233167803-a5869ba9-e287-4e22-9558-42c85ca7c38e.png)
![image](https://user-images.githubusercontent.com/90643392/233168263-2f2c7d15-0e83-4a07-b548-8a53aaa7474b.png)
